simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/ThreadExports.java

```diff
@@ -0,0 +1,99 @@
+package io.prometheus.client.hotspot;
+
+import io.prometheus.client.Collector;
+
+import java.lang.management.ManagementFactory;
+import java.lang.management.MemoryMXBean;
+import java.lang.management.MemoryPoolMXBean;
+import java.lang.management.MemoryUsage;
+import java.lang.management.ThreadInfo;
+import java.lang.management.ThreadMXBean;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Exports metrics about JVM thread areas.
+ * <p>
+ * Example usage:
+ * <pre>
+ * {@code
+ *   new ThreadExports().register();
+ * }
+ * </pre>
+ * Example metrics being exported:
+ * <pre>
+ *   jvm_threads_count{type="total"} 300
```

Comment:
This isn't consistent with the rest of the file, the _count suffix should be avoided outside of summaries/histograms, different labelnames is strongly discouraged and a total label such as this will cause problems.

I'd suggest removing this as we don't do this in other collectors.